lxd/api_project.go

```diff
@@ -339,6 +364,56 @@ func projectChange(d *Daemon, project *api.Project, req api.ProjectPut) response
 		return response.BadRequest(err)
 	}
 
+	if containerLimitChanged {
+		var names []string
+		err := d.cluster.Transaction(func(tx *db.ClusterTx) error {
+			var err error
+			names, err = tx.ContainerNames(project.Name)
```

Comment:
I'm deleting this function right now, so you should be avoid it.

I would suggest having a single bit of shared logic for both containers and vms. If either limit has changed, you should pull all instances for the project and iterate over them, looking at their types to count them as either container or vm.

Using `instanceLoadNodeProjectAll` instead, passing instancetype.Any to have both containers and VMs returned so you only need to hit the DB once even if both limits are changed.